Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM quay.io/keboola/docker-custom-python:2.0.4
 
+RUN mkdir -p /var/www && chown www-data:www-data /var/www
```

Comment:
Bezne bezia kontajnery na produkcii ako `www-data`, ... no v component portali sa da zapnut aby bezali vo vnutry kontajnera ako `root`. Pre `python-transformations-v2` to je zapnute lebo neslo instalovat balicky (`packages`), ale chceme sa toho zbavit. 

`/var/www` je home adresar pre uzivatela `www-data` a tam sa tie balicky instaluju, preto musi existovat a mat spravne nastavene `user:group`.